packages/donate-button-beta/src/components/widget/Frequency/index.tsx

```diff
@@ -0,0 +1,104 @@
+import cxs from 'cxs';
+import {useState} from 'preact/hooks';
+import {Fragment} from 'preact/jsx-runtime';
+import {COLORS} from 'src/components/widget/theme/colors.enum';
+
+const frequencyContainerCss = cxs({
+	display: 'flex'
+});
+
+const labelCss = cxs({
+	color: COLORS.Primary,
+	fontSize: '1rem',
+	lineHeight: 1.5,
+	fontWeight: 400,
+	letterSpacing: '-0.01em',
+	padding: '8px 0px',
+	flex: 1,
+	textAlign: 'center',
+	border: '1px solid',
+	borderColor: COLORS.LightGray,
+	transition: 'all .2s'
```

Comment:
We should avoid `transition: all` and instead specify what will be transitioned, limiting only to performant transitions.